packages/plugin-ext/src/plugin/env-variables-server-ext.ts

```diff
@@ -0,0 +1,46 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { RPCProtocol } from '../common/rpc-protocol';
+import { PLUGIN_RPC_CONTEXT, EnvMain } from '../common';
+import { EnvVariablesServer, EnvVariable } from '@theia/core/lib/common/env-variables';
+
+export class EnvVariablesServerExt implements EnvVariablesServer {
```

Comment:
We should not implement `EnvVariablesServer ` in the plugin host process. It was bad design when it was introduced. All these methods are not needed there.

The convention is to declare `EnvExt` next to `EnvMain` and only add necessary methods. Please clean it up. 🙏

Reply:
I did not create it, shall I remove it?

Reply:
yes, i think we should not implement `EnvVariablesServer`, but replace it with `EnvExt` shape defined next to `EnvMain` instead of adding unnecessary methods.

Reply:
OK. Let's do that.